packages/@react-aria/menu/src/useMenuTrigger.ts

```diff
@@ -91,13 +91,7 @@ export function useMenuTrigger(props: MenuTriggerAriaProps, state: MenuTriggerSt
     },
     menuProps: {
       ...overlayProps,
-      'aria-labelledby': menuTriggerId,
-      onMouseDown(e) {
```

Comment:
Appears to be unnecessary now.

Reply:
fixed by that usePress change to register?